ansible/files/steps/batch.sh

```diff
@@ -89,13 +90,14 @@ else
     if [ $EXTENSION = "json" ]; then
         # Executes Zeppelin notebook
         source activate zeppelin
-        zeppelin-execute -i ${NOTEBOOK_NAME}
+        zeppelin-execute -i ../${NOTEBOOK_NAME} -o ./note.json
         EXIT_CODE=$?
+        zeppelin-convert -i ../note.json -o ./${FILE_NAME}.md
         if [ $EXIT_CODE != 0 ]; then
-            zeppelin-convert -i note.json
+            cat ${FILE_NAME}.md
```

Comment:
If there is an error during the execution, the markdown will consist of the errors and it will get printed to the logs. If there are no errors during execution, nothing will be printed to the logs.